server/utils/tasks/table.ts

```diff
@@ -101,14 +101,79 @@ const unallocatedTableRows = (tasks: Array<Task>): Array<TableRow> => {
   return rows
 }
 
+const tasksTableRows = (tasks: Array<Task>, allocatedFilter: string): Array<TableRow> => {
+  const rows: Array<TableRow> =
+    allocatedFilter === 'allocated' ? allocatedTableRows(tasks) : unallocatedTableRows(tasks)
+
+  return rows
+}
+
+const allocatedTableHeader = () => {
+  return [
+    {
+      text: 'Person',
+    },
+    {
+      text: 'Days until due date',
+      attributes: {
+        'aria-sort': 'none',
+      },
+    },
```

Comment:
I think we still want to sort here, we can use the `sortHeader` helper here https://github.com/ministryofjustice/hmpps-approved-premises-ui/blob/79ca7f0d770e43ac5a5cb2cbe59769a365b802c1/server/utils/sortHeader.ts#L6. See https://github.com/ministryofjustice/hmpps-approved-premises-ui/blob/79ca7f0d770e43ac5a5cb2cbe59769a365b802c1/server/utils/placementRequests/table.ts#L125 for an example of usage

Reply:
@pezholio I'm not sure I cant find the field that accept for sortBy here in                                        override fun tasksReallocatableGet(
    type: String?,
    page: Int?,
    sortDirection: SortDirection?,
    allocatedFilter: AllocatedFilter?,
  )   override fun tasksReallocatableGet(
    type: String?,
    page: Int?,
    sortDirection: SortDirection?,
    allocatedFilter: AllocatedFilter?,
  )
like in placementRequestsDashboardGet

Reply:
Yeah, we need to add that. Will the tests allow you to pass it in, or do the tests fail?

Reply:
yah it will need SortFieldtype

Reply:
@pezholio I suggested adding the sort in a follow up PR once we have the API functionality